src/spoke/Spoke.sol

```diff
@@ -384,18 +384,29 @@ contract Spoke is Auth, Recoverable, ReentrancyProtection, ISpoke, IUpdateContra
         ShareClassDetails storage shareClass = _shareClass(poolId, scId);
         require(vaultFactory[factory], InvalidFactory());
 
-        // Deploy vault
         AssetIdKey memory assetIdKey = _idToAsset[assetId];
         IBaseVault vault = IVaultFactory(factory).newVault(
             poolId, scId, assetIdKey.asset, assetIdKey.tokenId, shareClass.shareToken, new address[](0)
```

Comment:
Curious @hieronx, do you know why we're passing here an empty array of wards? Is it dead code or do we expect some wards in the future?

Reply:
Generally it allows us to add wards in the future without swapping the vault factory. Seems potentially useful